src/picknik_ur_base_config/objectives/move_to_joint_state.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
-<root BTCPP_format="4" main_tree_to_execute="Move to Joint State">
-    <BehaviorTree ID="Move to Joint State" _description="Move to a specified joint state">
+<root BTCPP_format="4" main_tree_to_execute="MoveToJointState">
```

Comment:
Why did this need to be renamed? The `NoSpace` format is used for Behaviors and not Objectives, so I think we should change it back.